packages/common/components/search/marko.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
Remove both of these and call the browser components directly in the search page.